diff -up knavalbattle-20.03.90/src/networkdialog.cpp.omv~ knavalbattle-20.03.90/src/networkdialog.cpp
--- knavalbattle-20.03.90/src/networkdialog.cpp.omv~	2020-04-13 15:06:07.796200967 +0200
+++ knavalbattle-20.03.90/src/networkdialog.cpp	2020-04-13 15:06:12.070249202 +0200
@@ -81,11 +81,7 @@ NetworkDialog::NetworkDialog(bool client
         tmpLayout = new QHBoxLayout;
         tmpLayout->addWidget(tmp);
         tmpLayout->addWidget(m_games, 1);
-#if QT_VERSION < QT_VERSION_CHECK(5, 15, 0)
         connect(m_games, static_cast<void (KComboBox::*)(int)>(&KComboBox::currentIndexChanged), this, &NetworkDialog::serviceSelected);
-#else
-        connect(m_games, static_cast<void (KComboBox::*)(int, const QString &)>(&KComboBox::currentIndexChanged) , this, &NetworkDialog::serviceSelected);
-#endif
         mainLayout->addItem(tmpLayout);
         
         QWidget* frame=new QWidget(main);
